package.json

```diff
@@ -8,19 +8,21 @@
     "dist"
   ],
   "scripts": {
-    "build": "rimraf dist && babel src -d dist",
-    "docs": "gulp docs",
+    "docs": "gulp",
+    "build": "npm run build:dist && npm run build:docs",
+    "build:dist": "rimraf dist && babel src -d dist",
+    "build:docs": "gulp docs",
     "clean": "while read line; do rm -rf $line; done < .gitignore",
-    "predeploy:docs": "npm run docs",
+    "predeploy:docs": "npm run build:docs",
     "deploy:docs": "gh-pages -d docs/build -m 'deploy docs [ci skip]'",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
     "lint:watch": "watch 'npm run lint --silent' docs gulp src test",
-    "prepublish": "npm run build",
+    "prepublish": "npm run build:dist",
     "release:major": "ta-script npm/release.sh major",
     "release:minor": "ta-script npm/release.sh minor",
     "release:patch": "ta-script npm/release.sh patch",
-    "start": "gulp",
+    "start": "npm run docs",
```

Comment:
It seems it may be clearer to remove the `docs` script and call `npm run gulp` here. Is there a reason one may also want to run `npm run docs` separately?